docs/gui/release_notes_de.md

```diff
@@ -4,6 +4,10 @@ Im Folgenden finden Sie die Release Notes für das User Management.
 
 Technische Details zu einem Release finden Sie im zugehörigen [Changelog](https://docs.cloudogu.com/de/docs/dogus/usermgt/CHANGELOG/).
 
+## Release 1.16.4-1
+* Auf der Nutzerimport-Vorschauseite sind nun 25 Einträge pro Seite statt wie vorher 8
```

Comment:
This reads weird "statt wie vorher 8". How about "statt der vorherigen 8 Einträge"?